dinoDetail.php

```diff
@@ -31,7 +34,8 @@
     <a href="index.php" class="backLink"><img src="assets/image/DinoFoot.svg" /> Back</a>
 </header>
 <main>
-    <?php echo $singleDinoDetail; ?>
+<?php echo DinoFinder\Classes\ViewHelpers\DinoViewHelper::displaySequentialDino($singleDinoDetail, $id, $dinoCount);?>
```

Comment:
Should be indented

Reply:
When passing in $dinoCount it may be worth using intVal(). I think that the variable is returned as a string type- but if this works anyway, then it may not be necessary